### PR TITLE
Bump release stack version: support for the dev version

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Enable `bumpUtils.getNextMinorReleaseFor7()` when running the bump script.

## Why is it important?

The version with the next minor release was removed when refactoring the source of truth with the versions.

### CODE REVIEW

File: .ci/bumpStackReleaseVersion.groovy [hunk 1]
```diff
@@ -99,25 +99,26 @@ def prepareArguments(Map args = [:]){
   def assign = args.get('assign', '')
   def reviewer = args.get('reviewer', '')
   // TODO: to be adapted to the 8 releases, so maybe the branch should be the one driving this dynamically.
-  def stackVersion = bumpUtils.getCurrentMinorReleaseFor7()
-  def message = """### What \n Bump stack version with the latest release. \n ### Further details \n ${stackVersion}"""
+  def stackCurrentVersion = bumpUtils.getCurrentMinorReleaseFor7()
+  def stackCurrentMinorVersion = bumpUtils.getNextMinorReleaseFor7()
```
Comment (cachedout) [inline .ci/bumpStackReleaseVersion.groovy:103]: This is a bit odd? We get the current version by asking for the current minor but we get the current minor by asking for the next minor? Perhaps it's just the naming but this seems very confusing...

Comment (v1v) [reply .ci/bumpStackReleaseVersion.groovy:103]: You are right, `stackCurrentMinorVersion` is misleading :/ I'll fix it now

Comment (v1v) [reply .ci/bumpStackReleaseVersion.groovy:103]:
```suggestion
  def stackNextMinorVersion = bumpUtils.getNextMinorReleaseFor7()
```

File: .ci/bumpStackReleaseVersion.groovy [hunk 2]
```diff
@@ -99,25 +99,26 @@ def prepareArguments(Map args = [:]){
...
+  def message = """### What \n Bump stack version with the latest release. \n ### Further details \n ${stackCurrentVersion}"""
```
Comment (v1v) [inline .ci/bumpStackReleaseVersion.groovy:104]:
```suggestion
  def message = """### What \n Bump stack version with the latest release. \n ### Further details \n ${stackCurrentVersion} ${stackNextMinorVersion}"""
```

File: .ci/bumpStackReleaseVersion.groovy [hunk 3]
```diff
@@ -99,25 +99,26 @@ def prepareArguments(Map args = [:]){
...
   log(level: 'INFO', text: "prepareArguments(repo: ${repo}, branch: ${branch}, scriptFile: ${scriptFile}, labels: '${labels}', title: '${title}', assign: '${assign}', reviewer: '${reviewer}')")
   if (labels.trim() && !labels.contains('automation')) {
     labels = "automation,${labels}"
   }
-  return [repo: repo, branchName: branch, title: "${title} ${stackVersion}", labels: labels, scriptFile: scriptFile, stackVersion: stackVersion,
-          message: message, assign: assign, reviewer: reviewer]
+  return [repo: repo, branchName: branch, title: "${title} ${stackCurrentVersion} ${stackCurrentMinorVersion}", labels: labels, scriptFile: scriptFile,
+          stackCurrentVersion: stackCurrentVersion, stackCurrentMinorVersion: stackCurrentMinorVersion, message: message, assign: assign, reviewer: reviewer]
```
Comment (v1v) [inline .ci/bumpStackReleaseVersion.groovy:110]:
```suggestion
  return [repo: repo, branchName: branch, title: "${title} ${stackCurrentVersion} ${stackNextMinorVersion}", labels: labels, scriptFile: scriptFile,
          stackCurrentVersion: stackCurrentVersion, stackNextMinorVersion: stackNextMinorVersion, message: message, assign: assign, reviewer: reviewer]
```

File: .ci/bumpStackReleaseVersion.groovy [hunk 4]
```diff
@@ -99,25 +99,26 @@ def prepareArguments(Map args = [:]){
...
 }
 
 def createPullRequest(Map args = [:]) {
   bumpUtils.prepareContext(org: env.ORG_NAME, repo: args.repo, branchName: args.branchName)
 
   bumpUtils.createBranch(prefix: 'update-stack-version', suffix: args.branchName)
 
-  sh(script: "${args.scriptFile} '${args.stackVersion}' ''", label: "Prepare changes for ${args.repo}")
+  sh(script: "${args.scriptFile} '${args.stackCurrentVersion}' '${args.stackCurrentMinorVersion}'", label: "Prepare changes for ${args.repo}")
```
Comment (v1v) [inline .ci/bumpStackReleaseVersion.groovy:118]:
```suggestion
  sh(script: "${args.scriptFile} '${args.stackCurrentVersion}' '${args.stackNextMinorVersion}'", label: "Prepare changes for ${args.repo}")
```

File: .ci/bumpStackReleaseVersion.groovy [hunk 5]
```diff
@@ -129,8 +130,8 @@ def createPullRequest(Map args = [:]) {
   }
 
   // In case the docker image is not available yet, let's skip the PR automation.
-  if (!bumpUtils.isVersionAvailable(args.stackVersion)) {
-    log(level: 'INFO', text: "Version '${args.stackVersion}' is not available yet.")
+  if (!bumpUtils.isVersionAvailable(args.stackCurrentVersion) || !bumpUtils.isVersionAvailable(args.stackCurrentMinorVersion)) {
+    log(level: 'INFO', text: "Versions '${args.stackCurrentVersion}'/'${args.stackCurrentMinorVersion}' are not available yet.")
```
Comment (v1v) [inline .ci/bumpStackReleaseVersion.groovy:134]:
```suggestion
  if (!bumpUtils.isVersionAvailable(args.stackCurrentVersion) || !bumpUtils.isVersionAvailable(args.stackNextMinorVersion)) {
    log(level: 'INFO', text: "Versions '${args.stackCurrentVersion}'/'${args.stackNextMinorVersion}' are not available yet.")
```

**Review by cachedout — APPROVED** [review]
Thanks for fixing up the naming! 👍

**Pull request merged**